src/main/java/org/mskcc/oncokb/curation/OncokbCurationApp.java

```diff
@@ -139,6 +143,16 @@ public void importInitialCdxData() throws IOException {
         }
     }
 
+    @PostConstruct
+    public void importTranscriptFlags() {
+        Collection<String> activeProfiles = Arrays.asList(
+            env.getActiveProfiles().length == 0 ? env.getDefaultProfiles() : env.getActiveProfiles()
+        );
+        if (activeProfiles.contains("transcript-flag-importer")) {
+            canonicalTranscriptFlagImporter.importCanonicalTranscripts();
+        }
+    }
+
```

Comment:
Couldn't you have all this logic inside the importer itself?

Reply:
Yeah I don't see why not - could also extract the logic to check if profile is active into a util too. I just copied the cdxImporter.